addons/web/static/src/core/l10n/localization_service.js

```diff
@@ -7,14 +7,9 @@ import { localization } from "./localization";
 import { translatedTerms, _t } from "./translation";
 import { session } from "@web/session";
 
-const { config } = owl;
-
 export const localizationService = {
     dependencies: ["user"],
     start: async (env, { user }) => {
-        // add "data-toolip" to the list of translatable attributes in owl templates
-        config.translatableAttributes.push("data-tooltip");
```

Comment:
check why/if we need to pass the complete list of translatable attrs in the config (see start.js)

Reply:
waiting for @ged-odoo to update owl